src/datatypes/numeric.js

```diff
@@ -24,7 +24,14 @@ function generateFunctions(bufferReader,bufferWriter,size)
     };
   };
   var writer=function(value, buffer, offset) {
-    buffer[bufferWriter](value, offset);
+    try {
```

Comment:
For debugging info, the `debug` function should be used.

I still don't see what gain you have from logging every single number. Structures and Arrays could use some logging to get more context, but numbers ? A try/catch gets a small performance penalty because the function containing it cannot be optimized by v8, so it's a good idea to avoid putting it everywhere.

What would be a good idea is adding context to the Errors inside of Array and Structure readers/writers, so we know exactly which fieldInfo caused the trouble